web-registry/src/components/organisms/BuyCreditsModal/BuyCreditsModal.tsx

```diff
@@ -195,26 +194,24 @@ const BuyCreditsModal: React.FC<BuyCreditsModalProps> = ({
           validate={validationHandler}
           onSubmit={async (values, { setSubmitting }) => {
             setSubmitting(true);
-            if (location.pathname.includes('projects')) {
-              const track_data: any = {
-                price: selectedSellOrder?.askAmount,
-                batchDenom: selectedSellOrder?.batchDenom,
-                projectName: project.name,
-                creditClassName: project.id.split('-')[0],
-              };
-              if (values.retirementAction === 'manual') {
-                track_data['amountTradable'] = getFormattedNumber(
-                  values.creditCount,
-                );
-                track_data['amountRetired'] = getFormattedNumber(0);
-              } else if (values.retirementAction === 'autoretire') {
-                track_data['amountTradable'] = getFormattedNumber(
-                  values.creditCount,
-                );
-                track_data['amountRetired'] = getFormattedNumber(0);
-              }
-              track('buy2', track_data);
+            const track_data: any = {
+              price: selectedSellOrder?.askAmount,
+              batchDenom: selectedSellOrder?.batchDenom,
+              projectName: project.name,
+              creditClassName: project.id.split('-')[0],
+            };
+            if (values.retirementAction === 'manual') {
+              track_data['amountTradable'] = getFormattedNumber(
+                values.creditCount,
+              );
+              track_data['amountRetired'] = getFormattedNumber(0);
+            } else if (values.retirementAction === 'autoretire') {
+              track_data['amountTradable'] = getFormattedNumber(
+                values.creditCount,
```

Comment:
good catch! i had a reason for this, and i guess i never actually made i needed here. will try to fix that

Reply:
@victordelval i think it's best to just remove this code in this PR since it clearly was not working as intended, and address it as a part of this issue: https://github.com/regen-network/regen-registry/issues/1341

that's what i've done as part of this commit: e985c1d4